app/helpers/sellers_helper.rb

```diff
@@ -5,36 +5,114 @@ def self.generate_seller_json(seller:)
     locations = seller.locations
     seller = seller.as_json
     seller['locations'] = locations.as_json
-    seller['amount_raised'] = SellersHelper.calculate_amount_raised(
+    gift_card_amount = SellersHelper.calculate_gift_card_amount(
       seller_id: seller['id']
     )
+    donation_amount = SellersHelper.calculate_donation_amount(
```

Comment:
Instead of calling `SellersHelper.calculate` you could prob make the `self.` methods into private methods